### ISSUE TITLE
French translation : error in the emoji key link

### ISSUE THREAD

**Issue by jeanphibaconnais** [all-contributors/all-contributors#562]
**Describe the bug**
In the french documentation, the "emoji keys" link is broken.

**To Reproduce**
1. Go to https://allcontributors.org/docs/fr/overview
2. Scroll down to 'Et ensuite'
3. See error. `[Clé de Emoji (Référence de Types de Contribution)](clé d'emoji)`is display and not a link

**Expected behavior**
A correct link to the page

**Screenshots**
![image](https://user-images.githubusercontent.com/32639372/142575830-bdd14b69-de14-4315-b330-66c766a43fd8.png)

**Comment by Berkmann18** [comment]
Is that only on this page and only for the French translation?

**Comment by jeanphibaconnais** [comment]
Yes, i checked the others pages and French translation is the only one concerned by the bug

**Comment by Berkmann18** [comment]
I see I did some tweaks to https://crowdin.com/translate/all-contributors/10/en-fr?filter=basic&value=18, so hopefully, that should be fixed in the next update.
